x-pack/plugins/security_solution/public/management/hooks/artifacts/use_host_isolation_exceptions_access.test.tsx

```diff
@@ -68,7 +68,7 @@ describe('useHostIsolationExceptionsAccess', () => {
 
     await waitFor(() => {
       expect(checkArtifactHasData).not.toHaveBeenCalled();
-      expect(result.current).toBe(true);
+      expect(result.current.hasAccessToHostIsolationExceptions).toBe(true);
```

Comment:
i think a test case for `isHostIsolationExceptionsAccessLoading: true` would be useful, both here and in `policy_details.test.tsx`

Reply:
Agree!